ushahidi/platform-client-mzima
# [Outreachy Task Submission] Fixing NODE_ENV script issues for Windows

# Introduction
This fixes #789 
Windows isn't very compliant with the UNIX way of setting environment variables, and thus certain scripts that require setting environment variables will fail.

## How to test this PR
**Note**: You will need a Windows computer.
1. Navigate to your project root (`[your-directory]/platform-client-mzima/`) in your command line on a Windows PC.
2. Run `npm run mobile:android`.
3. Notice that you will see a `'NODE_ENV' is not recognized as an internal or external command...` error.
4. Checkout to this PR.
5. Repeat step 2.
6. This time around, it works.

## Extra comments:
I used the `cross-env` library, and updated the scripts in the `project.json` file that set the `NODE_ENV` in the `mobile-mzima-client` directory.
